vscode-build/vscode.patch

```diff
@@ -38,59 +32,27 @@ index 9e26dfeeb6e..00000000000
--					<iframe sandbox="allow-scripts allow-forms allow-same-origin"></iframe>
-+					<iframe sandbox="allow-scripts allow-forms allow-same-origin allow-popups"></iframe>
- 				</div>
- 
- 				<script src="${mainJs}" nonce="${nonce}"></script>
 diff --git a/package.json b/package.json
-index 2bca5115cac..53884a9bb17 100644
+index 2103fe1fe1a..92954953615 100644
 --- a/package.json
 +++ b/package.json
-@@ -44,15 +44,15 @@
-     "valid-layers-check": "node build/lib/layersChecker.js",
-     "update-distro": "node build/npm/update-distro.js",
-     "web": "node resources/web/code-web.js",
--    "compile-web": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js compile-web",
--    "watch-web": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js watch-web",
-+    "compile-web": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js compile-web",
-+    "watch-web": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js watch-web",
-     "eslint": "node build/eslint",
-     "playwright-install": "node build/azure-pipelines/common/installPlaywright.js",
--    "compile-build": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js compile-build",
--    "compile-extensions-build": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js compile-extensions-build",
--    "minify-vscode": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js minify-vscode",
--    "minify-vscode-reh": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js minify-vscode-reh",
--    "minify-vscode-reh-web": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js minify-vscode-reh-web",
-+    "compile-build": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js compile-build",
-+    "compile-extensions-build": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js compile-extensions-build",
-+    "minify-vscode": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js minify-vscode",
-+    "minify-vscode-reh": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js minify-vscode-reh",
-+    "minify-vscode-reh-web": "node --max_old_space_size=4095 $NODE_OPENSSL_OPTION ./node_modules/gulp/bin/gulp.js minify-vscode-reh-web",
-     "hygiene": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js hygiene",
-     "core-ci": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js core-ci",
-     "extensions-ci": "node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js extensions-ci"
-@@ -77,6 +77,7 @@
-     "spdlog": "^0.13.0",
-     "sudo-prompt": "9.2.1",
-     "tas-client-umd": "0.1.4",
+@@ -96,10 +96,10 @@
+     "kerberos": "^2.0.1",
+     "minimist": "^1.2.6",
+     "native-is-elevated": "0.7.0",
+-    "native-keymap": "^3.3.5",
```

Comment:
I removed everything involving `native-keymap` as that was causing issues with the build involving Electron (which we obviously don't care about)